pandas/io/parsers.py

```diff
@@ -71,14 +72,6 @@
     By file-like object, we refer to objects with a ``read()`` method, such as
     a file handler (e.g. via builtin ``open`` function) or ``StringIO``.
 %s
```

Comment:
I think it'd be better to name all the `%s`. Having them positional ames things a bit difficult to follow, at least for me.